decidim-core/app/controllers/decidim/errors_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ErrorsController < ApplicationController
+    authorize_resource :error_pages, class: false
```

Comment:
Yeah, I'm 👍 on skipping any extra logic, I'd be OK even with inheriting from `ActionController::Base`.